nemo/collections/llm/gpt/model/base.py

```diff
@@ -124,6 +132,8 @@ class GPTConfig(TransformerConfig, io.IOMixin):
     seq_length: int = 1024
     attention_softmax_in_fp32: bool = False
     masked_softmax_fusion: bool = True
+    cross_entropy_loss_fusion: bool = True
```

Comment:
I am unable to see the usage of this ?

Reply:
When we compute the loss this enables the use of a fused kernel:
https://github.com/NVIDIA/Megatron-LM/blob/5ec1e292822a5df9206bc04d371497b5d0e4ed4c/megatron/core/models/common/language_module/language_module.py#L38
While has better perf and uses less memory